### PR TITLE
use free variable for native fetch

### PR DESCRIPTION
**Pull request by Rich-Harris**
i have ABSOLUTELY NO IDEA how this passed CI in the first place, but apparently #5041 broke a bunch of stuff somehow? my browser suddenly started complaining that you can't call `fetch` if it's a property of something that isn't `window`. seems like a weird constraint but who am i to judge.

no changeset because #5041 wasn't released yet, so no need

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

**Comment by mrkishi** [comment]
Huh. Are the failures on Safari/Firefox? We were fooled by green checks pretty hard.

File: packages/kit/src/runtime/client/client.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ import {
 	notifiable_store,
 	scroll_state
 } from './utils.js';
-import * as fetcher from './fetcher.js';
+import { increment, decrement, initial_fetch, native_fetch } from './fetcher.js';
```
Comment (mrkishi) [inline packages/kit/src/runtime/client/client.js:14]: Maybe renaming `increment` and `decrement` to something else would make it clearer, at least locally if not in `fetcher.js`?

```suggestion
import { increment as lock_fetch, decrement as unlock_fetch, initial_fetch, native_fetch } from './fetcher.js';
```

**Comment by Rich-Harris** [comment]
No, just running `pnpm test` locally. Truly weird

**Comment by Conduitry** [comment]
Somewhat recently, there was a change to have Playwright use whatever version of Chrome it could find on your system, rather than using separate downloaded binaries. I wonder whether that could be related to the difference in behavior in the tests?

I'd be in favor of reverting that change anyway. Requiring the user have Chrome installed on their system makes local dev less portable, and not using a specific version makes tests less reproducible.

File: packages/kit/src/runtime/client/client.js [hunk 2]
```diff
@@ -613,10 +613,10 @@ export function create_client({ target, session, base, trailing_slash }) {
 
 			if (import.meta.env.DEV) {
 				try {
-					fetcher.increment();
+					increment();
```
Comment (mrkishi) [inline packages/kit/src/runtime/client/client.js:616]: Following previous comment:
```suggestion
					lock_fetch();
```

File: packages/kit/src/runtime/client/client.js [hunk 3]
```diff
@@ -613,10 +613,10 @@ export function create_client({ target, session, base, trailing_slash }) {
...
 					loaded = await module.load.call(null, load_input);
 				} finally {
-					fetcher.decrement();
+					decrement();
```
Comment (mrkishi) [inline packages/kit/src/runtime/client/client.js:619]: Continuing:
```suggestion
					unlock_fetch();
```

**Review by mrkishi — APPROVED** [review]

**Comment by sartoshi-foot-dao** [comment]
My latest local builds failing for perhaps the same reason:

```
> Using @sveltejs/adapter-vercel
✘ [ERROR] Could not resolve "@sveltejs/kit/install-fetch"

    .svelte-kit/vercel-tmp/serverless.js:1:29:
      1 │ import { installFetch } from '@sveltejs/kit/install-fetch';
        ╵                              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  The path "./install-fetch" is not exported by package "@sveltejs/kit":

    node_modules/@sveltejs/kit/package.json:47:13:
      47 │   "exports": {
         ╵              ^

  You can mark the path "@sveltejs/kit/install-fetch" as external to exclude it from the bundle,
  which will remove this error.

> Build failed with 1 error:
.svelte-kit/vercel-tmp/serverless.js:1:29: ERROR: Could not resolve "@sveltejs/kit/install-fetch"
```

Hope this helps.

**Pull request merged**